README.md

```diff
@@ -1966,6 +1966,24 @@ authClient.tokenManager.on('error', function (err) {
 });
 ```
 
+**NOTE**: If you manage the tokens state outside of this SDK, subscribe to `expired` event published by the `tokenManager` to get tokens properly renewed .
+
+```javascript
+// autoRenew set to true
+authClient.tokenManager.on('expired', function(key, expiredToken) {
+  // retrieves a new valid token with tokenManager.get
+  const token = await authClient.tokenManager.get(key);
```

Comment:
This reinforces the behavior we're hoping to get rid of - that a "get" will actually do a renewal behind the scenes.  Is there a way to have them kick that fetch off explicitly so we aren't encouraging code we intend to break soon?

Reply:
Updated to use `renew`

> `tokenManager.renew(key)`
Manually renew a token before it expires and update the stored value.